rustls/src/msgs/ech_test.rs

```diff
@@ -0,0 +1,16 @@
+use base64;
+use crate::msgs::codec::{Codec, Reader};
+use crate::msgs::handshake::ECHConfigs;
+
+#[test]
+fn test_echconfig_serialization() {
+    // An ECHConfig record from Cloudflare for
+    let base64_esni = "AEf+CQBDABNjbG91ZGZsYXJlLWVzbmkuY29tACCD91Ovu3frIsjhFKo0I1fPd/a09nzKMrjC9GZV3NvrfQAgAAQAAQABAAAAAA==";
```

Comment:
Should we replace the `esni` with `ech` here?

Reply:
I think you're reading an old diff. Should be "let base64_echconfig = ..." now.